pydrive2/apiattr.py

```diff
@@ -80,7 +80,10 @@ def __repr__(self):
 
     def update(self, *args, **kwargs):
         """Overwritten method of dictionary."""
+        BAD_URL_PREFIX = "https://www.googleapis.comhttps:"
         for k, v in dict(*args, **kwargs).items():
+            if k == "downloadUrl" and v.startswith(BAD_URL_PREFIX):
+                v = v.replace(BAD_URL_PREFIX, "https://www.googleapis.com")
```

Comment:
Instead of replace, it would be more safe to cut only the prefix.

Reply:
Or, you can pass `count=1` to `replace()`.

Reply:
done!  thanks @skshetry .